src/Uno.UWP/Helpers/ProtocolActivation.cs

```diff
@@ -0,0 +1,74 @@
+﻿#if __WASM__
+using System;
+using System.Linq;
+using System.Web;
+
+namespace Uno.Helpers
+{
+	public static class ProtocolActivation
```

Comment:
I'm wondering if it would be better located in the AppManifest.js file and processed as a configuration instead. Ultimately, the `Package.appxmanifest` will be used from the UWP app to generate a json configuration file.

Something like:

```
var UnoAppManifest = {
    splashScreenImage: "Assets/SplashScreen.scale-200.png",
    splashScreenColor: "#fff",
    protocols: [ 
      { 
         "url": "web+myscheme", 
         "name":  "Can we handle web+myscheme links?"
      }
   ]
}
```

@carldebilly do you have an opinion there ?

Reply:
I am not sure that would be a good user experience, as when the app requests the protocol registration, the user gets the permission pop-up. If it were done automatically, this would happen during startup, which would be confusing. This way the developer can ask for registration when actually needed (when user is already in the app, maybe after some tutorial for example). Also in terms of localization of the message, it is a bit easier when it is part of the code.

Reply:
I see. Let's keep it that way for now then.